peer.go

```diff
@@ -1796,6 +1827,28 @@ out:
 			p.sendDoneQueue <- struct{}{}
 			peerLog.Tracef("%s: acked queuehandler", p)
 
+		case timeout := <-p.blockStallActivate:
+			peerLog.Debugf("Activating block stall timer (%v "+
+				"seconds) for: %v", timeout, p)
```

Comment:
The timeout variable is a `time.Duration` and the print message uses `%v` which means it will print `5s`.  In words, the extra `seconds` is redundant.